gxa/src/main/webapp/WEB-INF/jsp/tiles/views/search-results/bioentity-information.jsp

```diff
@@ -14,8 +14,8 @@
 <script>
```

Comment:
You can remove the `elvariable` lines as they are no longer used.

Reply:
Will do it.